posthog/models/error_tracking/error_tracking.py

```diff
@@ -155,3 +159,51 @@ class ErrorTrackingIssueFingerprint(models.Model):
 
     class Meta:
         constraints = [models.UniqueConstraint(fields=["team", "fingerprint"], name="unique fingerprint for team")]
+
+
+def resolve_fingerprints_for_issue(team_id: int, issue_id: str) -> list[str]:
```

Comment:
Any reason not to pass the whole `Issue` here, rather than a stringly-typed id? (particularly thinking since the issue id is actually a UUID)